packages/_private/react-demo/src/ThemeSwitch.tsx

```diff
@@ -1,9 +1,9 @@
 import { useState } from 'react';
-import { SynSwitch } from '@synergy-design-system/react';
-import type {
```

Comment:
And here we have a big problem: `@react` exports the react wrappers, which are called EXACTLY like their native counterparts, e.g. `export const SynSwitch:React.FC` and in `@components` `export class SynSwitch: LitElement`. This crashes and makes the usage unintuitive. Suggestion: Only reexport the events for all wrappers, not the other types from `@components`.